src/createAnimatedComponent/InlinePropManager.ts

```diff
@@ -154,9 +154,9 @@ export class InlinePropManager {
       const shareableViewDescriptors =
         this._inlinePropsViewDescriptors.shareableViewDescriptors;
 
-      const maybeViewRef = NativeReanimatedModule.native
-        ? undefined
-        : ({ items: new Set([this]) } as ViewRefSet<any>); // see makeViewsRefSet
+      const maybeViewRef = shouldBeUseWeb()
```

Comment:
No need to evaluate it here each time. Instead evaluate `IS_NATIVE` in top level of this file.